### DISCUSSION TITLE
Per screen taskbar position

### DISCUSSION THREAD

**Discussion by teeteeteeteetee** [valinet/ExplorerPatcher#684] [Ideas]
It would be really nice if we could change secondary taskbar's its position depending on the monitor for example I have got 3 monitors
![image](https://user-images.githubusercontent.com/62479501/149812435-5bb9457b-8249-4b0a-8c3c-710525ecc37f.png)
on the green square my taskbar is on the left which is great however on the blue square the taskbar is also on the left which I dont like and i would prefer putting it on the right position. If i switched the position to right, it would b the exact opposite for me.

It basically looks like this:
![image](https://user-images.githubusercontent.com/62479501/149812897-7df57652-4851-459b-928b-182f2d7f2cbc.png)

Settings:
![image](https://user-images.githubusercontent.com/62479501/149811499-262430a4-5dae-4493-a4f5-6badf40ae8a0.png)

ye
![image](https://user-images.githubusercontent.com/62479501/149811105-398958cc-31f8-4596-9bb5-68a069f63202.png)

**Comment by valinet** [comment]
Right click any taskbar, uncheck Lock all taskbars, drag to move it wherever you want. Been there since Windows 98. Same as in Windows 10 as well.

**Reply by teeteeteeteetee** [reply]
Oh I thought it would move both of the taskbars, my bad.. I apologize!

**Comment by Anixx** [comment]
>Been there since Windows 98.

Since Windows 95.
